packages/http-client-python/generator/pygen/codegen/templates/client_container.py.jinja2

```diff
@@ -2,7 +2,9 @@
 {% import 'operation_tools.jinja2' as op_tools %}
 {# actual template starts here #}
 # coding=utf-8
-{{ code_model.options['license_header'] }}
+{% if code_model.license_header %}
+{{ code_model.license_header }}
+{% endif %}
```

Comment:
I think we could remove the none check since `.license_header` alreadys have string value even if it is empty string "".

Reply:
seems black could not handle the blank line correctly and the result will cause lint error, so i kept the current logic.